plugins/native/module/vtk.module

```diff
@@ -3,6 +3,7 @@ NAME
 DESCRIPTION
   A VTK module for the native plugin
 DEPENDS
+  f3d::vtkextPrivate
```

Comment:
PRIVATE_DEPENDS ?

Reply:
unless we should more the vtkF3DGLTFImporter to public module if thats useful for other plugins maybe ?

Reply:
yes I think it's better. `vtkextPrivate` shouldn't be accessed by plugins anyway

Reply:
alright